estate/models/model_real_estate_property_offer.xml

```diff
@@ -53,4 +53,17 @@
         <field name="required">True</field>
         <field name="on_delete">restrict</field>
     </record>
+
+    <record id="action_real_estate_property_offer_accept_offer" model="ir.actions.server">
+        <field name="name">Accept the offer</field>
+        <field name="model_id" ref="estate.model_real_estate_property_offer"/>
+        <field name="state">code</field>
+        <field name="code"><![CDATA[
+    for offer in records:
+        for property_offer in offer['x_property_id']['x_offer_ids']:
```

Comment:
The Python ORM way of writing works for this reading case (I personally find it easier to read) 
```suggestion
        for property_offer in offer.x_property_id.x_offer_ids:
```